File: src/Reference.CSharp/GeoPoint.cs
using System;
using Newtonsoft.Json;

namespace Reference.CSharp
{
    /// <summary>
    /// Geo point
    /// </summary>
    public class GeoPoint
    {
        /// <summary>
        /// UUID
        /// </summary>
        [JsonProperty("uuid")]
        public Guid Uuid { get; set; }
        /// <summary>
        /// Latitude
        /// </summary>
        [JsonProperty("lat")]
        public double Latitude { get; set; }
        /// <summary>
        /// Longitude
        /// </summary>
        [JsonProperty("lon")]
        public double Longitude { get; set; }
    }
}

File: src/Reference.VB/GeoPoint.vb
Imports System
Imports Newtonsoft.Json


''' <summary>
''' Geo point
''' </summary>
Public Class GeoPoint
    ''' <summary>
    ''' UUID
    ''' </summary>
    <JsonProperty("uuid")>
    Public Property Uuid As Guid
    ''' <summary>
    ''' Latitude
    ''' </summary>
    <JsonProperty("lat")>
    Public Property Latitude As Double
    ''' <summary>
    ''' Longitude
    ''' </summary>
    <JsonProperty("lon")>
    Public Property Longitude As Double
End Class


